client/app/lib/commonviews/verifypasswordmodal.coffee

```diff
@@ -53,8 +53,8 @@ module.exports = class VerifyPasswordModal extends ContentModal
               password            :
                 name              : 'password'
                 cssClass          : 'line-with'
-                label             : 'Current Password'
-                placeholder       : 'Enter your current password'
+                label             : 'Your Password'
```

Comment:
we don't need an extra label, as we tell in the line above. Or you can remove the text above and put it here in the label i.e. `label : Please enter your password to continue:`

![img](http://s.sinanyasar.com/170306_131001.png)